core/primitives-core/src/config.rs

```diff
@@ -353,6 +361,9 @@ impl Default for ExtCostsConfig {
             keccak256_byte: SAFETY_MULTIPLIER * 7157035,
             keccak512_base: SAFETY_MULTIPLIER * 1937129412,
             keccak512_byte: SAFETY_MULTIPLIER * 12216567,
+            ripemd160_base: SAFETY_MULTIPLIER * 15136567500, // 10 x sha256_base
+            ripemd160_block: SAFETY_MULTIPLIER * 80391170 * 8, // 10 x sha256_byte x 8 bytes
+            ecrecover_base: SAFETY_MULTIPLIER * 75682837500, // 50 x sha256_base
```

Comment:
These should be behind a feature flag? Also how are they determined?

Reply:
Same question here :)

Reply:
I didn't know that they needed to be behind the feature flag. Which do you recommend? The idea is to remove `protocol_feature_evm` entirely. Where it is in the code now is where the old EVM logic is when it was part of core and not as a contract. A later PR will be created to remove it in full. 

The costs are WIP as we had issues with the estimator before. I think, they should be ok now that the feature flags were removed, possibly won't have that issue again. I can attempt to run it to get accurate results. But, the cost is currently based off of ETH's costs using their sha256 cost as a base. They should be overestimates.

Reply:
AFAIK this was resolved in Slack. EVM precompiles need to have their own feature flag.

Reply:
Unfortunately, we need to be precise with the fees, because if we are not and we underestimate them someone will be able to grind our blockchain and slow block production to the point where it causes issues. Eyeballing costs through what they are on Ethereum does not provide guarantees and so we cannot do it. I am trying to run param estimator right now to see if we can speed up review of this PR.

Reply:
Feature flag added in https://github.com/near/nearcore/pull/4380/commits/3f08a35fa65f1a24be2fb46f6d72ca1b5c587a3b

Can't speak to the values themselves, so I'll leave this conversation open.

Reply:
@nearmax Let me know how it goes either here or in Slack.